airbyte-commons/src/main/java/io/airbyte/commons/json/JsonSchemas.java

```diff
@@ -58,7 +58,8 @@ public class JsonSchemas {
    * @param jsonNode - a json object with children that contain types.
    */
   public static void mutateTypeToArrayStandard(final JsonNode jsonNode) {
-    if (jsonNode.get(JSON_SCHEMA_TYPE_KEY) != null && !jsonNode.get(JSON_SCHEMA_TYPE_KEY).isArray()) {
+    if (jsonNode.get(JSON_SCHEMA_TYPE_KEY) != null && !jsonNode.get(JSON_SCHEMA_TYPE_KEY)
```

Comment:
@colesnodgrass i think your IDE is using a shorter line length than our standard. can you make sure it is set to 150 ([docs](https://docs.airbyte.com/contributing-to-airbyte/code-style))

Reply:
We actually have two different settings. In the docs we use `150` but in the style-sheet we set [comment lines to 100](https://github.com/airbytehq/airbyte/blob/74792c133677175e0871f49f61b0401cad17eabe/tools/gradle/codestyle/java-google-style.xml#L85). Looks like for some reason the `100` length was also being used for non-comment lines.

We should probably increase the comment-line length to match the `150` value.